src/main/java/org/dependencytrack/resources/v1/OsvEcosytemResource.java

```diff
@@ -42,7 +43,7 @@ public class OsvEcosytemResource extends AlpineResource {
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(
-            value = "Returns a list of all ecosystems in OSV",
+            value = "Returns a list of all not-yet-selected ecosystems in OSV",
```

Comment:
I feel like this should either be a separate endpoint (e.g. `/api/v1/integration/osv/ecosystem/inactive`), or a different solution is needed. If the behavior of the existing endpoint is changed like this, it will be unnecessarily hard to work out which ecosystems exist overall.

Relying on this endpoint to do the de-duplication will also not prevent duplicate values from entering the system. So perhaps performing de-duplication when *saving* the property (e.g. [here](https://github.com/DependencyTrack/dependency-track/blob/4.7.1/src/main/java/org/dependencytrack/resources/v1/AbstractConfigPropertyResource.java#L57)) would be more fool-proof?

Reply:
Duplicate values should not be able to be selected by end-user also, and sent back to backend in configProperty. 
So, either we can write a new endpoint for this, in order to display only not-yet-selected-available ecosystems in UI or to do same we can do this subtraction of (ALL - SELECTED) in frontend logic itself?